action.yml

```diff
@@ -77,5 +79,9 @@ runs:
 
         # update the PR body to set versions if different than creation
         gh pr edit --body "${BODY}"
+        if [ -n "${{ inputs.label }}" ];
+        then
+          gh pr --label "${{ inputs.label }}"
```

Comment:
Does this return a non-zero exit code if the PR already has this label?

Also note the behavior here is such that if you remove the label, the next time this task runs it will add the label back.

Reply:
I think it will return 0 if the pr already exists, but it is not documented:
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/8550208/234941636-0e39a929-656b-48e4-8ee9-0b656d6c6adb.png">